Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+                Console.Write("Введите первый катет прямоугольного треугольника: ");
+                double kat1 = Convert.ToDouble(Console.ReadLine());
+
+                Console.Write("Введите второй катет прямоугольного треугольника: ");
+                double kat2 = Convert.ToDouble(Console.ReadLine());

```

Comment:
Не стоит использовать методы класса Convert без особых причин. Лучше так
```suggestion
                double kat1 = double.Parse(Console.ReadLine());

                Console.Write("Введите второй катет прямоугольного треугольника: ");
                double kat2 = double.Parse(Console.ReadLine());

```